Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN yum install epel-release -y && \
     yum remove epel-release -y && \
     yum clean all
 
-RUN pip install raven --upgrade && pip install django==1.11.2 psycopg2==2.7.3
+RUN pip install raven --upgrade && pip install django==1.11.2 psycopg2==2.7.3 djangorestframework
```

Comment:
I know nobody's asked for my review on this. But can we use a specific version of DRF? Without a version, it'll install the bleeding edge which I'm not sure if we're ready to deal with. Anyway, the dependencies are going to be latest.

Reply:
@dharmit Yea this is currently in POC stage we will need to fix that stuff.
Ideally we want to discuss the details. I just wanted something that works, and i was able to get that by doing a  `$ python manage.py runserver` manually.
@bamachrn @navidshaikh if you guys have any comments please do add as well.

Reply:
Let's get the things working and tested, later note down the versions of dependencies and update the Dockerfiles.